addons/payment_alipay/models/payment_acquirer.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from hashlib import md5
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    provider = fields.Selection(selection_add=[
+        ('alipay', 'Alipay')
+    ], ondelete={'alipay': 'set default'})
+    alipay_payment_method = fields.Selection([
+        ('express_checkout', 'Express Checkout (only for Chinese Merchant)'),
+        ('standard_checkout', 'Cross-border'),
+    ], string='Account', default='express_checkout',
+        help="* Cross-border: For the Overseas seller \n* Express Checkout: For the Chinese Seller")
+    alipay_merchant_partner_id = fields.Char(
+        string='Merchant Partner ID', required_if_provider='alipay', groups='base.group_system',
+        help='The Merchant Partner ID is used to ensure communications coming from Alipay are valid and secured.')
+    alipay_md5_signature_key = fields.Char(
+        string='MD5 Signature Key', required_if_provider='alipay', groups='base.group_system',
+        help="The MD5 private key is the 32-byte string which is composed of English letters and numbers.")
```

Comment:
```suggestion
    alipay_merchant_partner_id = fields.Char(
        string='Merchant Partner ID', required_if_provider='alipay', groups='base.group_system')
    alipay_md5_signature_key = fields.Char(
        string='MD5 Signature Key', required_if_provider='alipay', groups='base.group_system')
```
I don't find these helper texts to be useful at all. One would expect a string like "This parameter can be found here" in the context of an integration configuration.